api/graphql/schema.graphql

```diff
@@ -1164,7 +1170,7 @@ input GenericJobTitleBelongsToMany {
 }
 
 type Mutation {
-    createUser(user: CreateUserInput! @spread): User @create @guard @can(ability: "create")
+    createUser(user: CreateUserInput! @spread): User  @guard @can(ability: "create") @field(resolver: "CreateUser")
```

Comment:
Is it still possible to leave `@create` there? Or does it need to be reimplemented in CreateUsers?

Reply:
they are mutually exclusive
for validation exception -> @create ( default resolver would throw the exception) 
for custom exception -> @field(resolver: "CreateUser")  ( we use custom resolvers to manually throw the exception )

Reply:
Thanks Brinda!